### ISSUE TITLE
Dependency of multizone parameters to zone records

### ISSUE THREAD

**Issue by PMehrfeld** [RWTH-EBC/TEASER#573]
## What is the problem?

- From teaser web export the parameters `VAir`, `ABuilding` and `ASurTot` are filled with direct values. If the parameter `AZone` is directly change in the zone record, it does not influence the mulitzone parameter `ABuilding`.

## Why do we want to solve it?

- There are direct links to zone record parameters.

## How do we want to solve it?

- Insert a dependency calculation like: `sum(multizone.zoneParam[:].AZone)`

This is just a suggestion.

**Comment by mlauster** [comment]
Nice suggestion, but won't since these parameters are defined in records and the current Modelica compiler can't handle such dependencies. But I'm not sure for what you need ABuilding, could be just for information or for calculation of ventilation rates

**Comment by PMehrfeld** [comment]
At least for `ABuilding` it works (I have only a one zone model currently).
I don't know, where this parameter is used. In the whole AixLib I found it only in the examples and in its declaration in `PartialMultizone`. However, it seems as it is not passed through at any point.
I didn't check the other two parameter
@MichaMans: Do you maybe know more about this?

Anyway, my solution works in my simulation setup:
![image](https://user-images.githubusercontent.com/11769433/54883901-f545c100-4e6a-11e9-8fe6-c5eab8f7a222.png)

**Comment by MichaMans** [comment]
@PMehrfeld 
I'm fine with your solution if it works. But does it make sense only changing the zone area without effecting the other parameters? Because with a changed zone area, whether the wall areas needs to change too, or they were initially false? Or do i miss something here?

**Comment by PMehrfeld** [comment]
Yes, there remain interdependencies within the zone record. This was just a suggestion for the class one level higher. E.g. you change in the drop down the zoneParam record(s), you don't need to adapt the 3 parameters above. Furthermore, you could use `numZones = size(zoneParam, 1)`.

It is no magic or really important thing. Just a suggestion to print the dependency string (as long as there exist a clear dependency) instead of writing the value ;)

**Comment by MichaMans** [comment]
Sounds good for me!

**Comment by mlauster** [comment]
Did you check in pedantic mode and with a multizone setup? If this works, I am also fine.

**Comment by PMehrfeld** [comment]
Yes, it checked in pedantic mode with 2 zones.

**Comment by mlauster** [comment]
Crazy, ok, then I allow it! :-)
